kernel/device.c

```diff
@@ -32,6 +32,7 @@ static inline void device_pm_state_init(const struct device *dev)
 		.usage = ATOMIC_INIT(0),
 		.lock = Z_MUTEX_INITIALIZER(dev->pm->lock),
 		.condvar = Z_CONDVAR_INITIALIZER(dev->pm->condvar),
+		.atomic_flags = ATOMIC_INIT(dev->pm->atomic_flags),
```

Comment:
```suggestion
```

Reply:
Nope, atomic_flags has to be initialized otherwise these devices will no be marked as wakeup capable.

Reply:
So then, couldn't all these static initializers be moved to `Z_DEVICE_STATE_DEFINE`?